packages/edit-site/src/components/page-patterns/style.scss

```diff
@@ -97,6 +105,30 @@
 	.edit-site-patterns__pattern-lock-icon {
 		fill: currentcolor;
 	}
+
+	.dataviews-view-grid & {
+		display: -webkit-box;
+		-webkit-line-clamp: 2;
+		-webkit-box-orient: vertical;
+		white-space: normal;
+		padding-top: 3px;
+		align-self: start;
+
+		button.components-button.is-link {
+			-webkit-line-clamp: 2;
+			-webkit-box-orient: vertical;
+			display: -webkit-box;
+			white-space: normal;
+		}
+	}
+}
+
+.dataviews-all-actions-button.has-icon:not(.has-text) {
+	.dataviews-view-grid & {
+		height: 24px;
+		width: 24px;
+		min-width: 24px;
+	}
 }
```

Comment:
As I said in [this comment](https://github.com/WordPress/gutenberg/pull/65200#discussion_r1828693533), I'd like to avoid hard-coding this style and address it as a separate PR.

Reply:
This style is no longer necessary.